build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 group = 'com.dumbdogdiner'
 
-version = '1.4.6'
+version = '1.4.7'
```

Comment:
Bump to 1.5.0 as New features have been added (see https://semver.org)

Reply:
OK fineeee